projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import { jobData } from '../jobData';
+import { VdkOption } from '../vdkOptions/vdk_options';
+
+export interface IDeployJobDialogProps {
+    /**
+     * Current Job name
+     */
+    jobName: string;
+    /**
+     * Current Team name
+     */
+    jobTeam: string;
+    /**
+    * Current Job path
+    */
+    jobPath: string;
+}
+
+
+export default class DeployJobDialog extends Component<IDeployJobDialogProps> {
+    /**
+   * Returns a React component for rendering a Deploy menu.
+   *
+   * @param props - component properties
+   * @returns React component
+   */
+    constructor(props: IDeployJobDialogProps) {
+        super(props);
+    }
+    /**
+    * Renders a dialog for creating a new deployment of a Data Job.
+    *
+    * @returns React element
+    */
+    render(): React.ReactElement {
+        // TODO: switch the checkbox into radio buttons
+        return (
+            <>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobName">Job Name:</label>
+                    <input type="text" id="jobName" className='jp-vdk-input' placeholder={this.props.jobName} onChange={this.onNameChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobTeam">Job Team:</label>
+                    <input type="text" id="jobTeam" className='jp-vdk-input' placeholder={this.props.jobTeam} onChange={this.onTeamChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobPath">Job Path:</label>
+                    <input type="text" id="jobPath" className='jp-vdk-input' placeholder={this.props.jobPath} onChange={this.onPathChange} />
```

Comment:
I have created an issue about this problem a while ago. You can see https://github.com/vmware/versatile-data-kit/issues/1341 . I won't be looking at this problem for a while since it is not essential for the integration and will require a lot of effort to be implemented.